### PR TITLE
Pushing url updates to master

### PR DESCRIPTION
**Pull request by ga0451**

### CODE REVIEW

File: FoodAPI/AppAPI/urls.py [hunk 1]
```diff
@@ -2,17 +2,25 @@
 from . import views
 from rest_framework import routers
 
-
+"""
```
Comment (MaxH2424) [inline FoodAPI/AppAPI/urls.py]: Why are there comment blocks here?

File: FoodAPI/AppAPI/urls.py [hunk 2]
```diff
@@ -2,17 +2,25 @@
...
     path('Rest/Zip/<int:zipcode>/', views.restZip),
-    path('', include(router.urls))
+    path('Item/', views.itemList),
+    path('Item/ID/<int:pk>/', views.itemID),
+    path('Item/Rest/<int:restId>/', views.itemRest),
+    path('User/', views.userList),
+    path('User/ID/<str:pk>/', views.userID),
+    path('Rating/', views.ratingList),
+    path('Rating/Rest/<int:restId>/', views.ratingRest),
+    #path('', include(router.urls))
```
Comment (MaxH2424) [inline FoodAPI/AppAPI/urls.py]: You probably forgot to delete this.

File: FoodAPI/AppAPI/views.py [hunk 3]
```diff
@@ -11,16 +11,26 @@
 
 
 
-
+# List rests and post single rest
 @csrf_exempt
 def restList(request):
     if request.method == 'GET':
         restObj = Rest.objects.all()
```
Comment (Nadia313) [inline FoodAPI/AppAPI/views.py:18]: Add delete function

File: FoodAPI/AppAPI/urls.py (hunk 1 above)
Comment (ga0451) [reply FoodAPI/AppAPI/urls.py]: They were used to block out older code but I will remove it now.

File: FoodAPI/AppAPI/urls.py (hunk 2 above)
Comment (ga0451) [reply FoodAPI/AppAPI/urls.py]: Thats correct. Deleting it now.

File: FoodAPI/AppAPI/views.py (hunk 3 above)
Comment (ga0451) [reply FoodAPI/AppAPI/views.py:18]: Thats a great idea and will be added in a future commit.

**Pull request merged**